x_train: [444.151254874, 265.652920229, 1, 0, 0, 0,
          0, 0, 0, 444.151254874, 265.652920229, 1,
          453.713001542, 353.61225102, 1, 0, 0, 0,
          0, 0, 0, 453.713001542, 353.61225102, 1,
          39.9057006836, 501.173736572, 1, 0, 0, 0,
          0, 0, 0, 39.9057006836, 501.173736572, 1,
          219.53805542, 708.448913574, 1, 0, 0, 0,
          0, 0, 0, 219.53805542, 708.448913574, 1,
          530.371704102, 451.306335449, 1, 0, 0, 0,
          0, 0, 0, 530.371704102, 451.306335449, 1,
          313.398620605, 79.0848236084, 1, 0, 0, 0,
          0, 0, 0, 313.398620605, 79.0848236084, 1]
